segmented_sieve_of_eratosthenes/segmentedSieveOfEratosthenes.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <cmath>
+#include <cstring>
+#include <vector>
+using namespace std;
+
+vector<int> segmentedSieve(int segMax) {
+	vector<int> Prime; //contains prime numbers uptop segMax
+	//segMax is point till you want to find Primes. ALWAYS pow(10,x)
+	int rootSegMax = sqrt(segMax); //root of segMax
```

Comment:
Please follow standard C++ guidelines to name variables and functions throughout the code, use underscores to separate them rather than using `camelCase`

Reply:
@Monal5031 is it really necessary? I don't think it will add any value.. if u think otherwise let me know

Reply:
@manish1997 Sorry but we follow all the coding guidelines in this repository but we ignore many of them in `ChefLib`. This repo was made for learning purpose and well documented code is the best way to learn.

Reply:
@Monal5031 okay, I understand. but I am habitual of this name convention and I don't think changing these will help me learn anything or it will help anyone else in learning it a better way. No offense :) Please close this PR  & #494  :)